rust/src/mime/smtp.rs

```diff
@@ -216,7 +198,7 @@ fn mime_smtp_process_headers(ctx: &mut MimeStateSMTP) -> (u32, bool) {
         } else if mime::slice_equals_lowercase(&h.name, b"content-transfer-encoding") {
             if mime::slice_equals_lowercase(&h.value, b"base64") {
                 ctx.encoding = MimeSmtpEncoding::Base64;
-                ctx.decoder = Some(MimeBase64Decoder::new());
+                ctx.decoder = Some(base64::Decoder::new());
```

Comment:
@jasonish do we risk naming conflicts with rust for using `base64` ?

Reply:
We don't. I was worried about the same and Jason suggested it be done this way. There was already `ffi/base64.rs` which uses the rust `base64` crate. :)